lib/rucio/db/sqla/models.py

```diff
@@ -774,7 +774,8 @@ class RSEFileAssociation(BASE, ModelBase):
                    CheckConstraint('bytes IS NOT NULL', name='REPLICAS_SIZE_NN'),
                    CheckConstraint('lock_cnt IS NOT NULL', name='REPLICAS_LOCK_CNT_NN'),
                    Index('REPLICAS_TOMBSTONE_IDX', 'tombstone'),
-                   Index('REPLICAS_PATH_IDX', 'path', mysql_length=NAME_LENGTH))
+                   Index('REPLICAS_PATH_IDX', 'path', mysql_length=NAME_LENGTH),
+                   Index('REPLICAS_PK_IDX', 'scope', 'name', 'rse_id'))
```

Comment:
This seems a bit wrong, as it is already the primary key but it was the only fix to get rid of the error:
```
Failed to add the foreign key constraint. Missing index for constraint 'SOURCES_REPLICA_FK' in the referenced table 'replicas'") [SQL: u'ALTER TABLE sources ADD CONSTRAINT SOURCES_REPLICA_FK FOREIGN KEY(scope, name, rse_id) REFERENCES replicas (scope, name, rse_id)']
```
while applying the alembic upgrade 22d887e4ec0a

Reply:
Also it is causing now an error with oracle because there is already an index

Reply:
Fixed, was caused by a bug in a downgrade step.